### DISCUSSION TITLE
When I set videoCallEnabled false I cannot join conference

### DISCUSSION THREAD

**Discussion by nguyenquynh201** [ant-media/Ant-Media-Server#6142] [Q&A]
Because I don't use a camera, I set videoCallEnabled = false , but I have the problem of being disconnected, I tried it in my program and in your example, they all have the same problem, you can see the image below. i am using WebRTC-Sdk-Android
![image](https://github.com/ant-media/Ant-Media-Server/assets/80728170/1258e9b0-80b9-4c50-bf2a-5fbe63b52a17)

**Comment by Mohit-3196** [comment]
Hi @nguyenquynh201,
Let me check and get back to you.

Thank you.

**Reply by nguyenquynh201** [reply]
I realized the problem was because I didn't grant camera access so I couldn't connect to your server

**Reply by Mohit-3196** [reply]
I See.
Thank you for the update @nguyenquynh201 

Have a great day.
